b3/clients.py

```diff
@@ -1532,6 +1533,20 @@ def getByCID(self, cid):
                 return None
         return None
 
+    def escape_string(value, mapping=None):
+        if isinstance(value, unicode):
+            return _escape_unicode(value)
```

Comment:
Where is this `_escape_unicode ` function defined?

Reply:
oh I didn't push to tested code :/ I will fix it asap I am kind of new to these version control things sorry